artifacts/install/operator-artifacts/6-controller.yaml

```diff
@@ -15,6 +15,6 @@ spec:
     spec:
       containers:
         - name: wso2am-controller
-          image: pubudu/k8s-wso2am-operator:0.8.1
+          image: rakhitharr/wso2am-controller:v17
           imagePullPolicy: Always
       serviceAccountName: wso2am-pattern-1-svc-account
```

Comment:
Add a newline at the end of the file

Reply:
Do we normally use a user specific location?

Reply:
New line is added in the latest code
This image was used for pre-release testing. It has been updated to wso2 images in latest code.